internal/p2p/host.go

```diff
@@ -0,0 +1,10 @@
+package p2p
+
+import (
+	"github.com/libp2p/go-libp2p-core/host"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	"github.com/tendermint/tendermint/config"
+)
+
+func NewHost(conf *config.P2PConfig) (host.Host, error)        { return nil, nil }
```

Comment:
These could use doc comments, even if only for the intent.

YMMV but for placeholders I often make the body panic so it compiles but doesn't do anything.